app/sql.js

```diff
@@ -1612,6 +1617,7 @@ function updateConversation(data) {
     type = $type,
     members = $members,
     name = $name,
+    isApproved = $isApproved,
```

Comment:
do we need to have indexes in that value? If not, no need to extract it, we can just keep in the json content itself